google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/__init__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .metadata_constants import MetadataConstants
+from .tag_templates_constants import TagTemplateConstants
```

Comment:
from `.tag_templates_constants` import `TagTemplateConstants`

Change file name to `tag_template_constants` so follow the className convention?